stboot.go

```diff
@@ -33,7 +33,7 @@ import (
 
 const (
 	hostCfgDefault = "/etc/host_configuration.json"
-	hostCfgHelp = `Location of Host Configuration file.
+	hostCfgHelp    = `Location of Host Configuration file.
```

Comment:
Hi @LinusSchroeder,

this PR looks good to me, but the base commit is not up-to-date.
#50 did already fix all linting errors. Could you rebase your PR to the current main head?

thanks!